package.json

```diff
@@ -49,9 +49,10 @@
     "lint:js": "gulp lint-js",
     "lint:css": "gulp lint-css",
     "lint:webext": "cd src && web-ext lint",
-    "run:nightly": "cd src && web-ext run --firefox=\"/Applications/Firefox Nightly.app\"",
-    "run:beta": "cd src && web-ext run --firefox=\"/Applications/Firefox Beta.app\"",
-    "run:stable": "cd src && web-ext run --firefox=\"/Applications/Firefox Stable.app\"",
-    "run:esr": "cd src && web-ext run --firefox=\"/Applications/Firefox ESR.app\""
+    "run:nightly": "cd src && web-ext run --firefox=nightly",
+    "run:beta": "cd src && web-ext run --firefox=beta",
+    "run:stable": "cd src && web-ext run --firefox=firefox",
+    "run:esr": "cd src && web-ext run --firefox=\"/Applications/Firefox ESR.app\"",
+    "run:developer": "cd src && web-ext run --firefox=firefoxdeveloperedition"
```

Comment:
Sounds reasonable, thanks. But I would remove the last line because the Firefox Developer Edition is in fact the same as Firefox Beta and shouldn't require separate tests during the add-on's development.

Reply:
I did not know they are the same. Removed the additional line.